README.md

```diff
@@ -149,11 +149,34 @@ Comment.disable_tracking do
   comment.update_attributes(:title => "Test 3")
 end
 
-# globally disable all history tracking
+# disable tracking for comments by default
+Comment.disable_tracking
```

Comment:
Let's alias `disable_tracking` to `disable_tracking!` and document the latter as the recommended way to do this. So either

```ruby
Comment.disable_tracking!
```

or

```ruby
Comment.disable_tracking do
 ...
end
```